cli/command/service/ps.go

```diff
@@ -32,6 +33,7 @@ func newPsCommand(dockerCli *command.DockerCli) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
+	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
```

Comment:
ah right, I do think `Only display task IDs` would make it a little bit clearer indeed 👼